tutor/templates/build/openedx/Dockerfile

```diff
@@ -187,6 +187,13 @@ ENV REVISION_CFG /openedx/config/revisions.yml
 COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/
 COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/
 
+{% if OPENEDX_ATLAS_PULL %}
+# Support the OEP-58 proposal behind a feature flag until it's fully implemented
+RUN pip install openedx-atlas
```

Comment:
`openedx-atlas` is not being installed by default yet

Reply:
`openedx-atlas==0.5.0` is already installed on `master`: https://github.com/openedx/edx-platform/blob/master/requirements/edx/base.txt#L781

Do we still need this `pip install`?

Reply:
@shadinaif now `nightly` should be tracking `quince` and atlas is installed on `open-release/quince.master`:

- https://github.com/openedx/edx-platform/blob/open-release/quince.master/requirements/edx/base.txt#L766

Reply:
nice, thank you. Removed